certified-connectors/Secib/README.md

```diff
@@ -0,0 +1,81 @@
+# SECIB
+SECIB provides a REST API which can be used to manage your SECIB's data (such as your case files, your calendar, your documents, etc).
+
+The SECIB connector uses that REST API to allow SECIB's users to automate their workflow. For instance, you can trigger a worklflow when a new case file is created in SECIB neo and then process this case file before sending an email notification with the case file's information to other SECIB's users.
+
+
```

Comment:
@thomasBeckSecib Please remove extra spacing in all the artifacts.

Reply:
I removed extra spacing in all artifacts.